.github/workflows/release.yml

```diff
@@ -13,12 +13,140 @@ on:
   push:
     branches:
       - release
+  # TODO: remove pull-request trigger, only for development of prebuilt binaries publishing
+  pull_request:
+    branches:
+      - develop
 
 jobs:
+  build-rust-binaries:
+    strategy:
+      matrix:
+        include:
+          - os: macos-latest
+            target: x86_64-apple-darwin
+            platform_name: macos
+            suffix: ''
+          - os: windows-latest
+            target: x86_64-pc-windows-msvc
+            platform_name: win64
+            suffix: .exe
+          - os: ubuntu-latest
+            target: x86_64-unknown-linux-gnu
+            platform_name: linux
+            suffix: ''
+    runs-on: ${{ matrix.os }}
+    env:
+      FILE_NAME: optic_diff${{ matrix.suffix }}
+    steps:
+      - uses: actions/checkout@28c7f3d2b5162b5ddd3dfd9a45aa55eaf396478b # https://github.com/actions/checkout/commits/v2
+      - uses: actions/setup-node@44c9c187283081e4e88b54b0efad9e9d468165a4 # https://github.com/actions/setup-node/commits/v1
```

Comment:
same, most recent v1 sha is "56899e050abffc08c2b3b61f3ec6a79a9dc3223d # v1.4.4`

Reply:
Addressed with https://github.com/opticdev/optic/pull/405/commits/7909b8a7582395c45ecc5ec348bc50789d35e428